### ISSUE TITLE
Lang file & command alias

### ISSUE THREAD

**Issue by linkinkov** [Katrix-/HomeSweetHome#8]
How about lang file, because my players dont know english?
And if possible add /sethome alias for /home set

**Comment by yepidoodles** [comment]
I already suggested the /sethome alias #5

**Comment by linkinkov** [comment]
@yepidoodles me more need lang file, because i cant compile scala, error with maven repo

**Comment by Katrix** [comment]
Changing the text shown in the plugin was probably the only feature I dropped when I made 2.0. If I can find a sane way I'm happy with I might add it back in again.

Would also cover the command names and help text?

**Comment by linkinkov** [comment]
If it is not difficult can you share full project of plugin? Then lang file cant be impliment now, i will translate sources files.

**Comment by Katrix** [comment]
If you're talking about KatLib, it's located [here](https://github.com/Katrix-/KatLib). If not then I don't know what you're talking about. The repository has everything (except sbt) needed to compile it. Don't start anything until I have laid the groundwork for what needs to be done. The hard part is to figure out what to do for the formatting. Most likely what I'll do there is to leave it fixed. For example, if some text is red, it will stay red. If some part of the text is highlighted with a specific color, it will stay highlighted that way.

**Comment by Katrix** [comment]
Did some groundwork [here](https://github.com/Katrix-/KatLib/blob/develop/shared/src/main/resources/katlib/resources.properties). This is for the `/<pluginid here>` and `/<pluginid here> help` commands. For example `/homesweethome` and `/homesweethome help`. This does not localize command names (probably possible if needed) or the names of the command arguments (I can't do anything about those AFAIK. Ask Sponge).

Does that work for you?

**Comment by linkinkov** [comment]
Yes! Can you impliment: https://github.com/Katrix-/KatLib/blob/develop/shared/src/main/resources/katlib/resources.properties
but in home plugin? I dont care about formating.

**Comment by linkinkov** [comment]
@Katrix- Can you provide new dev build?

**Comment by Katrix** [comment]
Not yet. Want to make sure everything is working first. I'll make a post once I'm sure everything's working.

**Comment by linkinkov** [comment]
@Katrix- okay. Thank you! I will waiting

**Comment by Katrix** [comment]
Ok, you should be safe to create a pull request with a new lang file now.

**Comment by Katrix** [comment]
Any progress on the localization?

**Comment by linkinkov** [comment]
How to past translation? pull request?

**Comment by Katrix** [comment]
Yeah, just send a pull request

**Comment by Katrix** [comment]
I released 2.2.0 for now. I'll try to get another release out when this is ready.
